content/en/dora_metrics/setup/failures.md

```diff
@@ -87,27 +87,37 @@ The severity of the incident in the DORA Metrics product is based on the [incide
 
 ### Mapping PagerDuty services to Datadog services
 
-When an incident event is received for a specific [PagerDuty service][101], Datadog attempts to retrieve the related Datadog service and team from the [Service Catalog][102].
+When an incident event is received for a specific [PagerDuty service][101], Datadog attempts to retrieve the related Datadog service and team from any triggering [Datadog monitors][107] and from the [Service Catalog][102].
 
-The matching algorithm works in the following scenarios:
+The matching algorithm works in the following steps:
```

Comment:
Does the algorithm stop at the first matching condition or does it run through all the steps?

```suggestion
The matching algorithm works in the following order:
```

Reply:
This would make a really good diagram or decision tree. Consider submitting a request with [brand design]!(https://datadoghq.atlassian.net/jira/core/projects/BRAND/forms/form/direct/1/10002)!